From: Laurentiu Tudor <laurentiu.tudor@nxp.com>
Date: Thu, 8 Jun 2017 17:28:51 +0300
Subject: staging: fsl-mc: drop unused forward declaration

Git-commit: b5d5740a00ca26549b695d3c957bb14879a20952
Patch-mainline: v4.13-rc1
References: fate#326530,fate#326531,fate#326535,fate#326538,fate#326539

This forward declaration of "struct fsl_mc_resource" is of no use so
drop it.

Signed-off-by: Laurentiu Tudor <laurentiu.tudor@nxp.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Signed-off-by: Mian Yousaf Kaukab <yousaf.kaukab@suse.com>
---
 drivers/staging/fsl-mc/include/mc-sys.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/staging/fsl-mc/include/mc-sys.h b/drivers/staging/fsl-mc/include/mc-sys.h
index dca7f9084e05..b5203707344a 100644
--- a/drivers/staging/fsl-mc/include/mc-sys.h
+++ b/drivers/staging/fsl-mc/include/mc-sys.h
@@ -46,7 +46,6 @@
  */
 #define FSL_MC_IO_ATOMIC_CONTEXT_PORTAL	0x0001
 
-struct fsl_mc_resource;
 struct mc_command;
 
 /**
-- 
2.11.0

